src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.html

```diff
@@ -1,78 +1,87 @@
 <h1 mat-dialog-title>Select a Topic</h1>
 <mat-dialog-content>
   <p>
-    A topic is a short description of your {{ entryTypeMetadata.term }}. There are {{ isHostedEntry ? 'two' : 'three' }} types of topics
-    that you can choose from:
+    A topic is a short description of your {{ entryTypeMetadata.term }}. There {{ topicOptions.length === 1 ? 'is' : 'are' }}
+    {{ topicOptions.length }} type{{ topicOptions.length === 1 ? '' : 's' }} of topics that you can choose from:
```

Comment:
Super nitpick, when the length is one, do you get "There is 1 type of topics that you can choose from:"?